### ISSUE TITLE
关于dice loss

### ISSUE THREAD

**Issue by yukkii01** [junqiangchen/LUNA16-Lung-Nodule-Analysis-2016-Challenge#15]
您好，我复现了您的程序，觉得很有收获，只是有一个疑问就是您的loss = -dice，我觉得这里应该是1-dice。祝好。

**Comment by junqiangchen** [comment]
是的可以改成1-dice
